src/nomad_material_processing/vapor_deposition.py

```diff
@@ -315,8 +321,8 @@ class VaporDepositionStep(ActivityStep):
         repeats=True,
     )
     substrate = SubSection(
```

Comment:
Not sure what I think about `SubstrateSetup` as a name but if we want to go for that I would  change the attribute name to something like `substrate_setups`.
```suggestion
    substrate_setups = SubSection(
```